imei.sh

```
@@ -221,6 +232,7 @@
 OS_SHORT_CODENAME="$(lsb_release -sc)"
 OS_ARCH="$(uname -m)"
 GH_FILE_BASE="https://codeload.github.com"
+GL_FILE_BASE="https://gitlab.com"
```

Shellcheck

[warning] GL_FILE_BASE appears unused. Verify use (or export if used externally).

(SC2034)